ecommerce/extensions/basket/templatetags/basket_tags.py

```diff
@@ -0,0 +1,38 @@
+
+from decimal import Decimal
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.filter(name='calculate_tax')
+def calculate_tax(price_excl_tax):
+    """
+    Calculate the tax that is going to be applied to a product price
+
+    Example:
+        if settings.TAX_RATE is 0.5 and price_excl_tax is 100, this will return 50 (0.5 * 100)
+
+    Arguments:
+        price_excl_tax: Price excluding tax
+    """
+    return round(Decimal(price_excl_tax) * Decimal(settings.TAX_RATE), 2)
+
+
+@register.simple_tag
+def get_tax_rate():
+    tax_rate_as_percentage = Decimal(settings.TAX_RATE) * Decimal(100.0)
+    return round(tax_rate_as_percentage, 2)
+
+
+@register.simple_tag
+def calculate_tax_of_order_line(price_excl_tax, price_incl_tax):
+    return price_incl_tax - price_excl_tax
+
+
+@register.simple_tag
+def calculate_vat_from_unit_price(unit_price_excl_tax, unit_price_incl_tax):
```

Comment:
@alfredchavez, nit: missing docstring

Reply:
@lgp171188 thanks, I've just added the missing docstrings :)